src/app/courses/courses.component.html

```diff
@@ -1,32 +1,62 @@
 <div class="mat-elevation-z8 courses">
   <p>{{message}}</p>
-  <a mat-raised-button routerLink="add" color="primary" routerLinkActive="active" i18n>Add Course</a>
-  <a mat-raised-button routerLink="request" color="primary" routerLinkActive="active" i18n>Request Course</a>
-  <h2 class="header" i18n> Courses </h2>
-  <div class="search-header">
-    <mat-form-field>
+  
+  <mat-toolbar>
+    <button mat-icon-button (click)="goBack()"><mat-icon>arrow_back</mat-icon></button>
+    <span i18n>Courses</span>
+    <span class="toolbar-fill"></span>
+    <mat-icon>search</mat-icon>
+    <mat-form-field class="font-size-1">
       <input matInput (keyup)="searchFilter($event.target.value)" i18n-placeholder placeholder="Search">
     </mat-form-field>
+  </mat-toolbar>
+
+  <div class="space-container">
+    <mat-toolbar class="primary-color font-size-1">
+      <span class="toolbar-fill"></span>
+      <button mat-button routerLink="request" routerLinkActive="active" [disabled]="!selection.selected.length">
+        <mat-icon aria-hidden="true" class="margin-lr-3" >compare_arrows</mat-icon>
+        <span i18n>Request Courses</span>
+      </button>
+      <button mat-mini-fab routerLink="add" ><mat-icon>add</mat-icon></button>
+    </mat-toolbar>
+  
+    <mat-table #table [dataSource]="courses" matSort>
+      <ng-container matColumnDef="select">
+        <mat-header-cell *matHeaderCellDef>
+          <mat-checkbox (change)="$event ? masterToggle() : null"
+          [checked]="selection.hasValue() && isAllSelected()"
+          [indeterminate]="selection.hasValue() && !isAllSelected()">
+        </mat-checkbox>
+        </mat-header-cell>
+        <mat-cell *matCellDef="let row">
+          <mat-checkbox (click)="$event.stopPropagation()"
+            (change)="$event ? selection.toggle(row) : null"
+            [checked]="selection.isSelected(row)">
+          </mat-checkbox>
+        </mat-cell>
+      </ng-container>
+
+      <ng-container matColumnDef="title">
+        <mat-header-cell *matHeaderCellDef mat-sort-header i18n> Title </mat-header-cell>
+        <mat-cell class="km-course-title" *matCellDef="let element"> {{element.courseTitle}} </mat-cell>
+      </ng-container>
+      <ng-container matColumnDef="action">
+        <mat-header-cell *matHeaderCellDef i18n> Action </mat-header-cell>
```

Comment:
Instead of this action , we can create menu as same as done in resource section.